native/packages/react-native-bpk-theming/src/BpkThemeProvider.js

```diff
@@ -16,11 +16,70 @@
  * limitations under the License.
  */
 
- import { ThemeProvider, withTheme } from 'theming';
+ import React from 'react';
+ import PropTypes from 'prop-types';
+ import { ThemeProvider } from 'theming';
+
+ // This is the same as https://github.com/iamstarkov/theming/blob/master/src/create-theme-listener.js
+ // but returns null instead of an error when context[CHANNEL] is undefined.
+ const createThemeListener = () => {
+   const CHANNEL = '__THEMING__';
+   const contextTypes = {
+     [CHANNEL]: PropTypes.object,
+   };
+
+   function initial(context) {
+     return context[CHANNEL] ? context[CHANNEL].getState() : null;
+   }
+
+   function subscribe(context, cb) {
+     return context[CHANNEL] ? context[CHANNEL].subscribe(cb) : null;
+   }
+
+   return {
+     contextTypes,
+     initial,
+     subscribe,
+   };
+ };
+
+ // This is the same as https://github.com/iamstarkov/theming/blob/master/src/create-with-theme.js
+ // but using the custom themeListener from above.
+ const createWithTheme = () => {
+   const themeListener = createThemeListener();
```

Comment:
In `theming` the pass a channel to the themeListener. Should we also be doing this?

Reply:
I copied it from the original library, but I can change it sure.

Reply:
So, I looked into it and it just always gets defaulted to `__THEMING__`. So I decided to just hard code that in, instead.